app/translations/ru.json

```diff
@@ -915,6 +915,7 @@
   "app.containers.SignUp.verify": "Подтвердить",
   "app.containers.SignUp.sendAnotherCode": "Отправить другой код",
   "app.containers.SignUp.eosActivePrivateKey": "Telos Active Private Key",
+  "app.containers.SignUp.eosPrivateKey": "Private Key",
```

Comment:
change to russian

Reply:
changed: https://github.com/peeranha/peeranha-web/pull/672/commits/8dc9684b33b7ee5f077cca58d021ebe0b5df0630